server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -212,6 +212,9 @@ private void operation(String cmd)
       case MetadataOperationType.SET_STORAGE_LEVEL_TO_MTREE:
```

Comment:
```suggestion
      case MetadataOperationType.SET_STORAGE_GROUP_TO_MTREE:
```

Reply:
@qiaojialin Do you want to modify all STORAGE_LEVEL to STORAGE_GROUP? If in that case, the author needs to check all naming.

Reply:
yes